ui/components/app/permissions-connect-header/permissions-connect-header.component.js

```diff
@@ -27,6 +27,7 @@ export default class PermissionsConnectHeader extends Component {
     ///: BEGIN:ONLY_INCLUDE_IN(flask)
     snapVersion: PropTypes.string,
     isSnapInstall: PropTypes.bool,
+    isSnapUpdate: PropTypes.bool,
```

Comment:
Do we need two props for this? Could we just rename the existing one?

Reply:
Yeah, true.